subprojects/test-runner/client/src/main/kotlin/com/avito/runner/scheduler/listener/ArtifactsTestListener.kt

```diff
@@ -111,6 +115,27 @@ internal class ArtifactsTestListener(
             }
         }
     }
-}
 
-private const val LOGCAT_LAST_LINES = 300
```

Comment:
@sboishtyan experimentally fetch full logcat buffer
was ~1.5mb in my local runs